services/web/pkg/service/v0/service.go

```diff
@@ -165,7 +164,7 @@ func (p Web) Static(f fs.FS, root string, ttl int) http.HandlerFunc {
 			return
 		}
 
-		w.Header().Set("Cache-Control", fmt.Sprintf("max-age=%s", strconv.Itoa(ttl)))
+		w.Header().Set("Cache-Control", "max-age="+strconv.Itoa(ttl))
```

Comment:
fmt.Sprintf is slower compared to string concatenation

Reply:
Yes.